### ISSUE TITLE
Notification bar for new content.

### ISSUE THREAD

**Issue by AnshuMishra17** [mage2tv/issues#45]
I think it will be nice feature to add a notification bar whenever there is a new content available.
which will directly links to the new content and open the collapsible.
![screen shot 2018-10-29 at 10 26 04 pm](https://user-images.githubusercontent.com/3141554/47666860-9edd9a80-dbca-11e8-9ffb-7c36b1e7999b.png)
Clicking on Fundamentals will open the Fundamentals and clicking on Plugins will open Fundamentals and Plugins and take user to that element.

**Comment by Vinai** [comment]
Thanks for opening the issue!
It's a great idea. It still needs be figured out though how to handle notifications for videos in different categories, because we often publish new content in different categories within the same batch.
The same might occur when many videos where published since the last login.
We could display a notification bar for each category that contains new videos. But then, once a notification was clicked, all others are lost.
Do you have a suggestion how that might be taken care of?

**Comment by AnshuMishra17** [comment]
Alternatively, we can just the notification bar and links to the new tag filter.
There are already some tags and we can add another tag with the name new. 
New tag can be assigned to the videos which are uploaded in the latest batch.
New tag from the existing videos will be removed automatically as soon as next batch is uploaded.
Or
There can be a new section, which will contain all the videos from the latest uploaded batch.

**Comment by Vinai** [comment]
The latest batch is already viewable using the date published filter. Would a „new“ tag add additional value?

**Comment by AnshuMishra17** [comment]
Yes, date published filter solves the purpose. But I think highlighted new tag or notification bar seems to be more user friendly to give user's the information that there is some content.
Not sure how to implement separate notification bar for each category :(

**Comment by Vinai** [comment]
How about adding a `new` tag to all videos that where released since the previous login?

**Comment by AnshuMishra17** [comment]
Yes, I think new tag will be good.
But it will for the videos that were released since previous login or the videos released in the latest batch.
Not sure which one will be better and if there is any difference between the two.
